Apps/W1/HybridGP/app/Permissions/HybridGPObjects.PermissionSet.al

```diff
@@ -91,5 +91,7 @@ permissionset 4029 "HybridGP - Objects"
                     table "GP Company Migration Settings" = X,
                     table "GP Migration Errors" = X,
                     table "GP Segment Name" = X,
-                    table "GP Company Additional Settings" = X;
+                    table "GP Company Additional Settings" = X,
+                    table GPSY40100 = X,
```

Comment:
Minor - Could we name these tables with space - e.g. "GP SY40100"?  Makes it easier to read and we will align them to the other tables.